### PR TITLE
Fix for issue #118

### PR DESCRIPTION
**Pull request by raghavendrabankapur**
Including the code to accept BodyAsFile in the response when the mappings are placed in the file.

### CODE REVIEW

**Comment by raghavendrabankapur** [comment]
@StefH I have included the code for the fix #118 . I have run the tests too and they are all working fine.

**Comment by raghavendrabankapur** [comment]
@stefH should I include any tests for this or this is fine?

**Comment by raghavendrabankapur** [comment]
@StefH I have added a test to cover the code. It required to fix one test too.

File: test/WireMock.Net.Tests/FluentMockServerTests.cs [hunk 1]
```diff
@@ -76,6 +76,34 @@ public void FluentMockServer_ReadStaticMapping_WithGuidFilename()
...
+        public void FluentMockServer_ReadStaticMapping_WithResponseBodyFromFile()
+        {
+            string guid = "00000002-ee28-4f29-ae63-1ac9b0802d87";
+
+            string folder = Path.Combine(GetCurrentFolder(), "__admin", "mappings", guid + ".json");
+            string json = File.ReadAllText(folder);
+
+            string responseBodyFilePath = Path.Combine(GetCurrentFolder(), "ResponseBodyFiles", "responsebody.json");
+
+            dynamic jsonObj = Newtonsoft.Json.JsonConvert.DeserializeObject(json);
```
Comment (StefH) [inline test/WireMock.Net.Tests/FluentMockServerTests.cs]: Newtonsoft.Json is not needed here. Please import this namespace in this file.

Comment (raghavendrabankapur) [reply test/WireMock.Net.Tests/FluentMockServerTests.cs]: I have moved this to using block and updated the commit. @StefH

**Comment by raghavendrabankapur** [comment]
@stefH updated the pull request.

**Comment by raghavendrabankapur** [comment]
@stefH please let me know if I can merge pull request?

**Comment by StefH** [comment]
I will merge this PR in the weekend. And create new NuGet.

**Comment by raghavendrabankapur** [comment]
@stefH please let me know if I can merge pull request?

**Pull request merged**